packages/prettier-plugin-cadl/ThirdPartyNotices.txt

```diff
@@ -8,7 +8,125 @@ original copyright notices and the licenses under which Microsoft received such
 components are set forth below. Microsoft reserves all rights not expressly
 granted herein, whether by implication, estoppel or otherwise.
 
-1. mkdirp version 1.0.4 (https://github.com/isaacs/node-mkdirp)
+1. ajv version 8.9.0 (ajv-validator/ajv)
```

Comment:
Is this file change expected? I when I fixed the tree shaking it started only pulling what was really used.

Reply:
I just blindly committed the output of the tool. Can't say for sure - it doesn't seem right to you?